docs/data-sources/vlan.md

```diff
@@ -18,9 +18,19 @@ data "netbox_vlan" "vlan1" {
   name = "vlan-1"
 }
 
-# Get VLAN by VLAN ID
+# Get VLAN by VID and IPAM role ID
 data "netbox_vlan" "vlan2" {
-  vid = 1234
+  vid  = 1234
+  role = netbox_ipam_role.example.id
+}
+
+# Get VLAN by name and tenant name
```

Comment:
Is this comment correct? It looks like it should filter by two things.

Generally: Now that we got rid of the AtLeastOneOf for vid/name, does searching for two attributes simultaneously in the examples still represent the common usecase?

This is the last thing in this MR I bother you with, I swear :D

Reply:
Yes, I believe the comment is correct, but I also agree that having `data "netbox_tenant"` in the example might easily be misinterpret. I'll change the example so that it only has `data "netbox_vlan"` and assumes that tenant information is available.

I'm not sure what the general use case might be. I can only confirm that my client needs to fetch VLAN information by searching using multiple attributes, because they have overlapping VLAN IDs, which can only be distinguished by tenant.

I also want to add that I'm happy to receive feedback from my PR. It's a sign of a proper active project.